### PR TITLE
CLOUD-327 document how to update

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: source/update.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   to ``1.1.0`` version should look as follows::
+
+     kubectl patch deployment percona-server-mongodb-operator \
+        -p'{"spec":{"template":{"spec":{"containers":[{"name":"percona-server-mongodb-operator","image":"percona/percona-server-mongodb-operator:1.1.0"}]}}}}'
+
+     kubectl patch default my-cluster-name --type=merge --patch '{
+        "spec": {"replsets":{ "image": "percona/percona-server-mongodb-operator:1.1.0-mongod4.0" },
+            "mongod": { "image": "percona/percona-server-mongodb-operator:1.1.0-mongod4.0" },
+            "backup":   { "image": "percona/percona-server-mongodb-operator:1.1.0-backup" }
+        }}'
```
Comment (dAdAbird) [inline source/update.rst]: patch command should be:
```
 kubectl patch psmdb my-cluster-name --type=merge --patch '{
    "spec": {
        "image": "percona/percona-server-mongodb-operator:1.1.0-mongod4.0",
        "backup": { "image": "percona/percona-server-mongodb-operator:1.1.0-backup" }
    }}'
```

the same below

File: source/update.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
+Update Percona Server for MongoDB Operator
+===========================================
+
+Starting from the version 1.1.0 Percona Kubernetes Operator for MongoDB allows
```
Comment (tduzan) [inline source/update.rst]: the Percona Kubernetes Operator

File: source/update.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Semi-automatic update
+---------------------
+
+#. Edit the ``deploy/cr.yaml`` file, setting ``updateStrategy`` key to
+   ``RollingUpdate``.
+
+#. Now you should `apply a patch <https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/>`_ to your
+   deployment, supplying necessary image names with a newer version tag. This
+   is done with the ``kubectl patch deployment`` command. For example, updating
+   to ``1.1.0`` version should look as follows::
```
Comment (tduzan) [inline source/update.rst]: to the 1.1.0 version

File: source/update.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Manual update
+-------------
+
+#. Edit the ``deploy/cr.yaml`` file, setting ``updateStrategy`` key to
+   ``OnDelete``.
+
+#. Now you should `apply a patch <https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/>`_ to your
+   deployment, supplying necessary image names with a newer version tag. This
+   is done with the ``kubectl patch deployment`` command. For example, updating
+   to ``1.1.0`` version should look as follows::
```
Comment (tduzan) [inline source/update.rst]: to the 1.1.0 version

**Pull request merged**